src/config/settings.js

```diff
@@ -135,4 +146,52 @@ function parseSaveError(err) {
   return friendlierMsg;
 }
 
+// updateTable is a hash of [integer, function(settings) => void].
```

Comment:
If we do this in the [TypeScript JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) way, some tooling may be able to read it.

```suggestion
// @type {Object<number, (typeof defaultSettings) => void>}
```
(Of course, we'd also need to convert this comment block to JSDoc syntax for that to work.)

Reply:
done.  I also used https://jsdoc.app/tags-type.html for the actual syntax to use.  I prefer more explicit notation like `Array.<Object<number, thing> => void>` over operator syntax.